File: typespec/problem/global-coordinator/authentication.go
package globalcoordinator

import "github.com/vetchium/src/typespec/problem"

var AuthenticationRequiredError = problem.Details{
	Type: "vetchium-problem-details/" +
		"global-coordinator-authentication-required",
	Title:  "Global coordinator authentication required",
	Status: 401,
	Detail: "A valid global coordinator bearer credential is required",
}

File: typespec/problem/global-coordinator/authentication.ts
import type { Details } from "../details.ts";

export const AuthenticationRequiredError: Readonly<Details> = {
  type: "vetchium-problem-details/global-coordinator-authentication-required",
  title: "Global coordinator authentication required",
  status: 401,
  detail: "A valid global coordinator bearer credential is required",
};
